superset/sql_lab.py

```diff
@@ -233,7 +233,7 @@ def execute_sql_statement(  # pylint: disable=too-many-arguments,too-many-statem
         )
     if apply_ctas:
         if not query.tmp_table_name:
-            start_dttm = datetime.fromtimestamp(query.start_time)
+            start_dttm = datetime.fromtimestamp(query.start_time / 1000)
```

Comment:
is query.start_time a value in milliseconds?  That appears to tbe the case as datetime.fromtimestamp expects values in seconds since epoch and you are dividing by 1000 here.  To make this code more clear for future devs I would recommend creating a variable MILLISECONDS_PER_SECOND = 1000
Then  change
start_dttm = datetime.fromtimestamp(query.start_time / MILLISECONDS_PER_SECOND)

This adds clarity to what the division is accomplishing and avoid the "[magic number](https://en.wikipedia.org/wiki/Magic_number_(programming))" scenario where future dev will ask what does 1000 represent?